pkg/util/util.go

```diff
@@ -165,7 +165,7 @@ func ExecCommand(name string, args ...string) string {
 	cmdOut, err := exec.Command(name, args...).Output()
 	if err != nil {
 		s := strings.Join(append([]string{name}, args...), " ")
-		klog.Errorf("error executing command %q: %v", s, err)
+		klog.Infof("error executing command %q: %v", s, err)
```

Comment:
ideally the error should bubble up instead.

Reply:
+1 to bubbling up the error as well

Reply:
ok

Reply:
@jichenjc I think bubbling up the error was meant to change the return values to be `(string, error)` instead of just `string` and let the caller decide what to do with the error, if any.